relay/relay_test.go

```diff
@@ -443,6 +443,13 @@ func (t *testRelaySuite) TestHandleEvent(c *C) {
 	c.Assert(pos, DeepEquals, binlogPos)
 	c.Assert(gs.String(), Equals, "") // no GTID sets in event yet
 
+	// after handle rotate envent, we save and flush the meta immediately
+	newCtx, newcancel := context.WithTimeout(context.Background(), 10*time.Millisecond)
```

Comment:
newCancel. please install a plugin to find naming problems

Reply:
Feels much better with this plugin: `Code Spell Checker`